index.test.js

```diff
@@ -216,6 +216,20 @@ describe('validate-commit-msg.js', function() {
       expect(errors).to.deep.equal(['INVALID COMMIT MSG: subject does not match subject pattern!']);
       expect(logs).to.deep.equal([msg]);
     });
+
+    it('should lowercase type when autoFix is true and make it valid', function() {
+      m.config.autoFix = true;
```

Comment:
May I ask why you are not in favour of `beforeEach` / `afterEach`? :)

I think in mocha it makes sense to use `describe`-global variables, as the `this` context in mocha does not give you easy access to the running test instance? But in jasmine it would make sense much more sense to use the `this` context, for example

```js
describe(function() {
  beforeEach(function() {
    this.prop = 'value';
  });

  it('...', function() {
    expect(this.prop).toBe('value');
  });
});
```

Also jasmine will clear all these instance variables, so they will be garbage collected!